JumpscaleLibs/clients/minio/MinioClient.py

```diff
@@ -0,0 +1,114 @@
+"""
+Minio Healing Client
+"""
+
+from Jumpscale import j
+import requests
+
+JSConfigClient = j.baseclasses.object_config
+
+
+class MinioClient(JSConfigClient):
+    """
+    Minio healing client object
+    """
+
+    _SCHEMATEXT = """
+        @url = jumpscale.minio.client
+        name** = "" (S)
+        url** = "" (S)
+        """
+
+    def get_jobs(self):
+        resp = requests.get(f"{self.url}/jobs")
+        resp.raise_for_status()
+        jobs = resp.json()
+    
+        minio_jobs = []
+        for job_id, job in jobs.items():
+            minio_job = MinioJob(
+                self.url, job_id, job["location"], job["type"], job["running"]
+            )
+            minio_jobs.append(minio_job)
+        return minio_jobs
+
+    def heal(self, background=False, file_path=""):
```

Comment:
I think u don't need inputs here. the file_path (I would call it `report_file` actually is ONLY needed for foreground jobs) hence. if report_file is provided then it must be a foreground job. hence I would change this to 
```python
def heal(self, report_file=None):
```
And then make sure it's documented that if report_file is provided it will start a foreground job.